### ISSUE TITLE
Mudbrick craft unbalanced

### ISSUE THREAD

**Issue by oversword** [obl3pplifp/BlS-issue-tracker#300]
Mud brick requires 4 adobe, which require 3 wheat (9 wheat => 3 straw) and 1 mud each
It has been suggested this is a little too expensive to craft

This is not an urgent issue, just a suggestion

**Comment by pl608** [comment]
what would be a good recipe for it? dirt+water(wet mud) --> furnace= mudbrick?

**Comment by oversword** [comment]
I don't think we should have to introduce a new item for it (wet mud), also using a water bucket in a craft makes it hard to craft large amounts, and mudbrick is a building block so people would want that.
Maybe it should just output 4 mud bricks for 4 adobe?

Maybe it shouldn't even use adobe, why not just use mud (swaz:mud)? I guess because the texture is more like abode...

Maybe the adobe recipe should be easier? Maybe 1mud + 1wheat = 1adobe, or maybe 1mud + 1straw block = 2adobe?

**Comment by oversword** [comment]
There's also weirdness where mud bricks are used to make adobe brick stairs even though you can cook mud bricks into swaz:brick_wall which has the same texture... maybe fix that while you're at it, they should all use the brick wall

**Comment by oversword** [comment]
Okay final decision:
1mud + 1straw block = 2adobe
4adobe = 4mud bricks
"Bricked Stair" variants should be crafted from "Brick Wall", not "Mud Brick"

These changes should be made in https://github.com/BlockySurvival/bls_custom
